README.md

```diff
@@ -23,40 +23,64 @@ First, follow [webpack's guide](https://webpack.js.org/configuration/devtool/) t
 Then, add the plugin to your webpack config:
 
 ```js
-const DevToolsIgnorePlugin = require('devtools-ignore-webpack-plugin')
+const DevToolsIgnorePlugin = require("devtools-ignore-webpack-plugin");
 
 module.exports = {
   // ...,
-  plugins: [
-    new DevToolsIgnorePlugin()
-  ]
-}
+  plugins: [new DevToolsIgnorePlugin()],
+};
 ```
 
 ## Options
 
-Currently, this plugin only supports one option: `shouldIgnorePath`. It is a function that checks whether a file should be ignored. The function takes a single argument, which is the path of the file. It should return a boolean value.
+Currently, this plugin supports two options: `shouldIgnorePath` and `potentialSourceMapExtension`.
+
+# shouldIgnorePath
```

Comment:
This should be `###`. Same for the new option.

Reply:
updated!